services/postprocessing/pkg/config/defaults/defaultconfig.go

```diff
@@ -28,6 +28,7 @@ func DefaultConfig() *config.Config {
 		},
 		Postprocessing: config.Postprocessing{
 			Events: config.Events{
+				Workers:  3,
```

Comment:
I thought you opted for `10`. Why not?

Reply:
as @rhafer mentioned: something has to give. The number of workers is configurable now and I wanted to start conservatively.